builtin/settingtypes.txt

```diff
@@ -77,8 +77,11 @@
 # Sections are marked by a single line in the format: [Section Name]
 # Sub-section are marked by adding * in front of the section name: [*Sub-section]
 # Sub-sub-sections have two * etc.
-# There shouldn't be too much settings per category; settings that shouldn't be
-#  modified by the "average user" should be in (sub-)categories called "Advanced".
+# There shouldn't be too many settings per category.
+#
+# The top-level category "Advanced" is handled specially and its contents only
+# shown when a checkbox is checked. It contains settings not for the "average user".
+# FIXME: some other categories also depend on show_advanced and I don't know why.
```

Comment:
This FIXME shouldn't b here. These other categories depending on show_advanced is intended, they should just be documented too.